### DISCUSSION TITLE
JSON training format ?

### DISCUSSION THREAD

**Discussion by vsraptor** [explosion/spaCy#6609] [Help: Other Questions] [answered]
Where can I find the description of the .json format used for training for NER tagging ?

**Comment by svlandeg** [comment, marked as answer]
You can find the details here: https://spacy.io/api/annotation#json-input

FYI - you can ask these general questions on the "Discussions" board instead of the Issue tracker, as we try to keep the latter focused mostly on bugs. I converted your issue to here :-)

**Comment by vsraptor** [comment]
got it thanks
